docs/source/demos/finnegans_wake_demo.rst

```diff
@@ -34,8 +34,16 @@ without revealing data keys to intermediary entities.
 Install NuCypher
 ----------------
 
-    Acquire the ``nucypher`` application code and install the dependencies.
-    For a full installation guide see the :doc:`/guides/installation_guide`.
+Acquire the ``nucypher`` application code and install the dependencies:
```

Comment:
Should we just link to the installation guide rather than reproducing the instructions here?

Reply:
I actually think that is current a source of confusion since running the demo requires a non-standard installation of nucypher and local copy of the database.